app/models/course/group.rb

```diff
@@ -27,8 +28,8 @@ def set_defaults
   # @return [Boolean]
   def should_create_manager?
     course && creator &&
-      course.course_users.exists?(user: creator) &&
-      !group_users.exists?(user: creator)
+      course.course_users.exists?(user_id: creator.id) &&
```

Comment:
This one is required cause we can't do `group_user.user` for now. 

There are two solutions: 
1) We add a `has_one :user, through: :course_user` in the `group_user` model.
2) We access `user` through course_user, maybe define a method?